src/views/self/view_self.cpp

```diff
@@ -7,10 +7,14 @@
 #include "util/scripts.hpp"
 #include "views/view.hpp"
 
+#include <fiber_pool.hpp>
 #include <script/globals/GPBD_FM_3.hpp>
+#include <thread_pool.hpp>
```

Comment:
These are includes from YimMenu source code, use `#include ""` instead of `#include <>`